.pullapprove.yml

```diff
@@ -9,5 +9,7 @@ reviewers:
   - battletoilet
   - wdo3650
   - mikewheaton
+  - dzearing
+  - cliffkoh
   name: pullapprove
-  required: 2
+  required: 1
```

Comment:
Why only 1 reviewer? We often have issues that aren't caught by the first reviewer.